builtin/game/falling.lua

```diff
@@ -1,6 +1,34 @@
 -- Minetest: builtin/item.lua
 
 local builtin_shared = ...
+local SCALE = 0.667
+
+local facedir_to_euler = {
```

Comment:
local pi = math.pi
will be a little faster

Reply:
Does not matter at all in this case because it's only run once during initialization.